.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,60 @@
+name: Babelfish for PostgreSQL Documentation
+description: Propose changes to Babelfish for PostgreSQL Documentation
+title: "[Documentation]: "
+labels: ["documentation", "untriaged"]
+assignees:
+  - gopinathpai
+  - nasbyj
+  - rcv-aws
+  - susanmdouglas-aws
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this documentation improvement.
```

Comment:
How about:

Thanks for taking the time to fill out this documentation improvement request.